.github/workflows/min_version_test.yml

```
@@ -36,7 +36,7 @@ jobs:
 
       - uses: subosito/flutter-action@44ac965b96f18d999802d4b807e3256d5a3f9fa1 # pin@v2.16.0
         with:
-          flutter-version: "3.0.0"
+          flutter-version: "3.13.0"
```

Let's try keeping this and see how it goes without adding the explicit dependency on `web`:
```suggestion
          flutter-version: "3.0.0"
```